x-pack/plugins/cases/public/components/actions/tags/translations.ts

```diff
@@ -51,3 +51,11 @@ export const SELECTED_TAGS = (selectedTags: number) =>
     defaultMessage: 'Selected: {selectedTags}',
     values: { selectedTags },
   });
+
+export const NO_TAGS_AVAILABLE = i18n.translate('xpack.cases.actions.tags.noTagsAvailable', {
+  defaultMessage: 'No tags available. Type to add one.',
```

Comment:
> @lcawl Any suggestions for the text? I think we should indicate to the users that to add a tag they have to search.

I think the piece of info I was missing until I tested this myself was "where" we're asking them to type. I suggest changing the string to something like this:

```suggestion
  defaultMessage: 'No tags available. To add a tag, type it in the search bar.',
```

Reply:
Thanks, @lcawl! Sorry, I should explain better.

Reply:
I received additional feedback from @gchaps recommending "enter" instead of "type":
```suggestion
  defaultMessage: 'No tags available. To add a tag, enter it in the query bar.',
```